components/config-mgmt-service/backend/postgres/schema/sql/01_noop.up.sql

```diff
@@ -0,0 +1 @@
+  
```

Comment:
Why is this needed?

Reply:
The whitespace change is a mistake, I had been writing the "CREATE TABLE" code in this file on accident.

The reason we have a noop migration file here in the first place is that I couldn't get all the migration tooling working (with hab &etc.) without having at least one file in this directory, so I needed to make a noop migration in the last PR which added postgres to the config management service.